.github/.OwlBot.yaml

```diff
@@ -0,0 +1,12 @@
+docker:
```

Comment:
Let's remove both of these owlbot files for now. I believe if we add them our automation might start kicking in and doing stuff.